http-builder-ng-core/src/main/java/groovyx/net/http/JavaHttpBuilder.java

```diff
@@ -335,6 +335,18 @@ public JavaHttpBuilder(final HttpObjectConfig config) {
         this.clientConfig = config.getClient();
         this.hostnameVerifier = config.getExecution().getHostnameVerifier();
         this.sslContext = config.getExecution().getSslContext();
+
+        if( clientConfig.getClientCustomizer() != null ){
+            throw new UnsupportedOperationException("Client implementation configuration is not available for the core Java implementation.");
+        }
+    }
+
+    /**
+     * The core Java client implementation does not support direct client access. This method will throw an {@link UnsupportedOperationException}.
+     */
+    @Override
+    public Object getClientImplementation() {
```

Comment:
I was able to get the customizer to work, but since we don't create the client until we use it, I left client access unsupported - we can probably refactor it to work if needed, but this should be good enough for now.